storybookjs/storybook
# Angular: Add global CSF3 renderer

Issue: https://github.com/storybookjs/storybook/issues/15498

## What I did

Added a basic global renderer for Angular and CSF3
Enabled previewCsfV3 for the angular-cli example storybook

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

Added a couple of stories to examples/angular-cli
